package/src/components/MessageList/MessageList.tsx

```diff
@@ -1151,33 +1143,14 @@ const MessageListWithContext = (props: MessageListPropsWithContext) => {
     }
   }
 
-  const ListFooterComponent = useCallback(
-    () => (
-      <View style={shouldApplyAndroidWorkaround ? styles.invertAndroid : undefined}>
-        <FooterComponent />
-      </View>
-    ),
-    [shouldApplyAndroidWorkaround, FooterComponent],
-  );
+  const ListFooterComponent = useCallback(() => <FooterComponent />, [FooterComponent]);
```

Comment:
Do we have to wrap these components in `useCallback()`? 
I'm pretty sure that
```tsx
const ListFooterComponent = FooterComponent;
```
will work the same, and would be more performant too.